component/authentication-endpoint/src/main/webapp/totp.jsp

```diff
@@ -71,6 +71,27 @@
         <script src="js/html5shiv.min.js"></script>
         <script src="js/respond.min.js"></script>
         <![endif]-->
+        <script>
+            // Handle form submission preventing double submission.
+            $(document).ready(function(){
+                $.fn.preventDoubleSubmission = function() {
+                    $(this).on('submit',function(e){
+                        var $form = $(this);
+                        if ($form.data('submitted') === true) {
+                            // Previously submitted - don't submit again
```

Comment:
End the comment with a fullstop

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-totp/commit/5055ed855a9116ac12b376422d3f0c526a8cd7d4